src/bot/commands/random/dad-jokes.ts

```diff
@@ -0,0 +1,60 @@
+import { Command, Input, Listener } from '@api';
+import * as request from 'request';
+import { Message } from 'discord.js';
+import { Emoji } from '@bot/libraries/emoji';
+import { Url } from 'url';
+import { monitorEventLoopDelay } from 'perf_hooks';
```

Comment:
This import and the one below it don't seem to be used in the code. (WebStorm would auto remove these xP) It doesn't really hurt anything much, but if we wanted to give people to option to pick and choose which modules / commands they want to incorporate in their version of the bot, it would be nice if they only had to download dependencies that they actually needed.